google-cloud-storage/samples/acceptance/files_test.rb

```diff
@@ -179,6 +190,15 @@ def mock_cipher.random_key
     end
   end
 
+  it "download_file_into_memory" do
+    bucket.create_file StringIO.new(file_content), remote_file_name
```

Comment:
Are we making sure that we're deleting this file at the end of tests?

Reply:
This is covered by the existing setup. The `after` block deletes all the files within a bucket.

https://github.com/googleapis/google-cloud-ruby/blob/2d4442b816f5b333cda560f01af61648260db72a/google-cloud-storage/samples/acceptance/files_test.rb#L73-L76

Reply:
Thanks.